server/core/store.go

```diff
@@ -703,3 +703,24 @@ func IsTiFlashStore(store *metapb.Store) bool {
 	}
 	return false
 }
+
+// IsEqualLabels returns whether have same store label
+func (s *StoreInfo) IsEqualLabels(labels []*metapb.StoreLabel) bool {
+	if len(s.GetLabels()) != len(labels) {
+		return false
+	}
+	storeLabels := s.GetLabels()
+	for _, label := range labels {
+		find := false
+		for _, slabel := range storeLabels {
+			if label.Key == slabel.Key && label.Value == slabel.Value {
+				find = true
+				break
+			}
+		}
```

Comment:
hard to read. maybe you can extract a new function or use `slice.AllOf`/`slice.NoneOf` to simplify.

Reply:
updated.